### DISCUSSION TITLE
About "qmap" package in R

### DISCUSSION THREAD

**Discussion by muhshif** [btschwertfeger/python-cmethods#28] [Q&A]
Now, this is a question out of my curiosity. By referring to what, you developed this package? Are you familiar with the bias correction "qmap" package of "R programming" already? Or is there any other library from python itself that inspired you?

**Comment by btschwertfeger** [comment]
During my studies, I had the opportunity to work in the "Dynamics of the Paleoclimate" section at the Alfred-Wegener-Institute, where I found a passion for bias corrections. However, my section lead expressed his frustration about the lack of usable tools for these corrections, and many scientists are forced to resort to alternative methods or even write their own code. As a software developer, I understand the challenges scientists face when trying to implement their ideas into code. Nevertheless, I found it relatively straightforward to convert the formulas from referenced publications into code.

I have limited experience with other tools for applying bias correction methods. While xclim is an option, it runs significantly slower than python-cmethods for comparable methods. BiasAdjustCXX is likely the fastest available option.

Since leaving AWI for a job in software development, I've continued to work on extensions for python-cmethods in my free time out of personal interest and with the hope that it can benefit someone's research. It's unfortunate that there seems to be a lack of transparency and accessibility around these methods, and it's my belief that everyone should have access to information on existing methods and the most efficient ways to apply them.
